ui/src/app/views/data-page/component.tsx

```diff
@@ -123,6 +124,8 @@ export const DataPage = withCurrentWorkspace()(class extends React.Component<
         conceptSetsApi().getConceptSetsInWorkspace(namespace, id),
         dataSetApi().getDataSetsInWorkspace(namespace, id)
       ]);
+      const existingDatasetName = dataSets.items.map(dataSet => dataSet.name);
```

Comment:
Nit: I'd avoid the intermediate variable, and just put the map inside the setState call.